verifier.sh

```diff
@@ -472,8 +472,14 @@ export NUMBA_DISABLE_JIT=1
 # run webui
 echo \$GEM_TOOLS_ONLY
 if [ -z \$GEM_TOOLS_ONLY ]; then
+    sudo mkdir -p /var/www	
+    sudo chown -R ubuntu /var/www	
     cd oq-engine/openquake/server
     cp local_settings.py.tools local_settings.py
+    pip install django-cookie-consent
+    python manage.py migrate
+    python manage.py loaddata fixtures/0001_cookie_consent.json
```

Comment:
Should be better if 0001_cookie_consent.json will be moved below standalone repository because it is already managed here